lib/ddr/models/has_admin_metadata.rb

```diff
@@ -61,7 +61,7 @@ module HasAdminMetadata
                predicate: Ddr::Vocab::Asset.workflowState,
                multiple: false
 
-      delegate :publish, :publish!, :unpublish, :unpublish!, :published?, to: :workflow
+      delegate :publish!, :unpublish!, :published?, :publishable?, to: :workflow
```

Comment:
@dchandekstark Yes.  :publish! and :unpublish! now do a lot more than just call :publish / :unpublish on an object and then save it.  Seemed potentially confusing to have a public method :publish which just sets the workflow attribute to "published" and a :publish! method that does that (and saves the object) and then (by default) publishes everything underneath it.